core/src/main/scala/org/apache/spark/storage/BlockManagerStorageEndpoint.scala

```diff
@@ -27,17 +27,17 @@ import org.apache.spark.util.{ThreadUtils, Utils}
 
 /**
  * An RpcEndpoint to take commands from the master to execute options. For example,
- * this is used to remove blocks from the slave's BlockManager.
+ * this is used to remove blocks from the replica's BlockManager.
```

Comment:
Did you mean replica here? It does seem to work, but could also be "storage endpoint".